test/DacpacTool.Tests/ScriptParserTests.cs

```diff
@@ -0,0 +1,164 @@
+using Microsoft.SqlTools.ServiceLayer.BatchParser;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NSubstitute;
+using Shouldly;
+
+namespace MSBuild.Sdk.SqlProj.DacpacTool.Tests
+{
+    [TestClass]
+    public class ScriptParserTests
+    {
+        private const string testProjectPath = "../../../../TestProjectWithPrePost";
```

Comment:
Perhaps we should rename this to TEST_PROJECT_PATH to match with the general C# guidelines.

Reply:
Done